packages/flutter/lib/src/material/dialog.dart

```diff
@@ -49,9 +49,12 @@ class Dialog extends StatelessWidget {
     this.elevation,
     this.insetAnimationDuration = const Duration(milliseconds: 100),
     this.insetAnimationCurve = Curves.decelerate,
+    this.insetPadding = const EdgeInsets.symmetric(horizontal: 40.0, vertical: 24.0),
+    this.clipBehavior = Clip.none,
     this.shape,
     this.child,
-  }) : super(key: key);
+  }) : assert(clipBehavior != null)
+     , super(key: key);
```

Comment:
I think this is typically formatted this way:
```dart
  }) : assert(clipBehavior != null), 
       super(key: key);
```

Reply:
I could have sworn I have seen it this way in other files, but looking at the style guidelines again, you are indeed correct. Thx. I will fix that shortly.